### PR TITLE
Fix some links

### PR DESCRIPTION
**Pull request by bebatut**
As mentioned in #710, some links are failing. I tried to fix some but I can not solve all of them.

The remaining errored links:

- In `topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html`
    - `https://www.bx.psu.edu/~anton/share/ng_test_data/var/raw_mother-ds-1.fq.gz failed: 404 No error`
    - `https://www.bx.psu.edu/~anton/share/ng_test_data/var/raw_mother-ds-2.fq.gz failed: 404 No error`

    ping @nekrut 

- In `topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html`: `https://doi.org/10.1080/STA-200066418 failed: got a time out (response code 303)`

   @shiltemann I tried to change with a direct link to the article but it does not change anything

- In `topics/proteomics/tutorials/protein-id-oms/tutorial.html`: `http://proteowizard.sourceforge.net/ failed: response code 0 means something's wrong`

    It looks like all site hosted by sourceforge.net are down

- In `topics/variant-analysis/tutorials/diploid-variant-calling/tutorial.html`: `http://snpeff.sourceforge.net/SnpEff.html failed: response code 0 means something's wrong`

    It looks like all site hosted by sourceforge.net are down

- In `topics/variant-analysis/tutorials/non-dip/tutorial.html`
    - `https://www.bx.psu.edu/~anton/share/ng_test_data/bmmb554/hw4/f.fq.gz failed: 404 No error`
    - `https://www.bx.psu.edu/~anton/share/ng_test_data/bmmb554/hw4/phix.fa failed: 404 No error`
    - `https://www.bx.psu.edu/~anton/share/ng_test_data/bmmb554/hw4/r.fq.gz failed: 404 No error`

    ping @nekrut

### CODE REVIEW

File: topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ We here proposed to re-analyze these data at least until genotypes determination
 
 ## Data upload
 
-The original data is available at [STACKS website](https://creskolab.uoregon.edu/stacks/tutorial/stacks_samples.tar.gz) and the subset used here is findable on [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs).
+The original data is available at [STACKS website](http://catchenlab.life.illinois.edu/stacks/param_tut.php) and the subset used here is findable on [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs).
```
Comment (nsoranzo) [inline topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md]: I can't find any data in either websites (the second just redirects to https://www.cesgo.org/en/ )

File: topics/proteomics/tutorials/protein-id-oms/tutorial.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ If your data were generated on a low resolution mass spectrometer, use ***PeakPi
...
->   > The vendor libraries used by MSConvert are only licensed for Windows systems and are therefore rarely implemented in Galaxy instances. If ***msconvert*** {% icon tool %} is not available in your Galaxy instance, please install the software on a Windows computer and run the conversion locally. You can find a detailed description of the necessary steps [here](http://genesis.ugent.be/files/costore/practicals/bioinformatics-for-proteomics/1-Peptide-and-Protein-Identification/1.2-Peak-List-Generation/1.2-Peak-List-Generation.pdf). Afterwards, upload the resulting mzML file to your Galaxy history.
+>   > The vendor libraries used by MSConvert are only licensed for Windows systems and are therefore rarely implemented in Galaxy instances. If ***msconvert*** {% icon tool %} is not available in your Galaxy instance, please install the software on a Windows computer and run the conversion locally. You can find a detailed description of the necessary steps on [here](https://compomics.com/bioinformatics-for-proteomics/identification/) (on "Peak List Generation"). Afterwards, upload the resulting mzML file to your Galaxy history.
```
Comment (nsoranzo) [inline topics/proteomics/tutorials/protein-id-oms/tutorial.md]: `on [here]`: remove `on`?

File: topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md (hunk 1 above)
Comment (bebatut) [reply topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md]: ping @yvanlebras

File: topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ tutorial_name: genetic-map-rad-seq
 # Introduction
 {:.no_toc}
 
-Original description is reachable on a [dedicated page of the official STACKS website](http://catchenlab.life.illinois.edu/stacks/tut_gar.php). Writers describe that they developed a genetic map in the spotted gar and present here data from a single linkage group. The gar genetic map is an F1 pseudotest cross between two parents and 94 of their F1 progeny. They took the markers that appeared in one of the linkage groups and worked backwards to provide the raw reads from all of the stacks contributing to that linkage group.
+Original description is reachable on a [dedicated page of the official STACKS website](http://catchenlab.life.illinois.edu/stacks). Writers describe that they developed a genetic map in the spotted gar and present here data from a single linkage group. The gar genetic map is an F1 pseudotest cross between two parents and 94 of their F1 progeny. They took the markers that appeared in one of the linkage groups and worked backwards to provide the raw reads from all of the stacks contributing to that linkage group.
```
[the remark this answers is not in the document]
Comment (bebatut) [reply topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md]: ping @yvanlebras

File: topics/proteomics/tutorials/protein-id-oms/tutorial.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ If your data were generated on a low resolution mass spectrometer, use ***PeakPi
...
 >
 >   > ### {% icon comment %} Comment: Local Use of MSConvert
->   > The vendor libraries used by MSConvert are only licensed for Windows systems and are therefore rarely implemented in Galaxy instances. If ***msconvert*** {% icon tool %} is not available in your Galaxy instance, please install the software on a Windows computer and run the conversion locally. You can find a detailed description of the necessary steps on [here](https://compomics.com/bioinformatics-for-proteomics/identification/) (on "Peak List Generation"). Afterwards, upload the resulting mzML file to your Galaxy history.
+>   > The vendor libraries used by MSConvert are only licensed for Windows systems and are therefore rarely implemented in Galaxy instances. If ***msconvert*** {% icon tool %} is not available in your Galaxy instance, please install the software on a Windows computer and run the conversion locally. You can find a detailed description of the necessary steps on [here](https://compomics.com/bioinformatics-for-proteomics/identification/) ("Peak List Generation"). Afterwards, upload the resulting mzML file to your Galaxy history.
```
Comment (nsoranzo) [inline topics/proteomics/tutorials/protein-id-oms/tutorial.md]: I think I didn't make myself clear, I meant remove `on` before `here`, which sounds a bit weird.

Comment (bebatut) [reply topics/proteomics/tutorials/protein-id-oms/tutorial.md]: 😄 
Fixed

**Comment by bebatut** [comment]
@nekrut @yvanlebras Can you have a look at the links? Thanks a lot!

**Comment by shiltemann** [comment]
anbody know a good alternative for the now-dead link https://bioteam.net/products/galaxy-appliance/ ?

**Comment by nsoranzo** [comment]
@shiltemann Just the bioteam homepage?

**Comment by yvanlebras** [comment]
Sorry, don't see previous notifications (as for the gtn paper ;) )... I will have a look ASAP!

**Comment by nekrut** [comment]
@bebatut - just pushed link fix. Sorry for delay.

**Comment by bgruening** [comment]
@bebatut I tried to resolve the conflict.

**Pull request merged**